src/status_im/contexts/profile/settings/events.cljs

```diff
@@ -70,6 +70,15 @@
                                        {:on-success on-success}])
              :on-cancel nil}]]})))
 
+(rf/reg-event-fx :profile.settings/toggle-sepolia-test-network
+ (fn [{:keys [db]}]
+   (let [value      (get-in db [:profile/profile :is-sepolia-enabled?])
+         on-success #(rf/dispatch [:wallet/initialize])]
+     {:fx [[:dispatch
+            [:profile.settings/profile-update :is-sepolia-enabled?
+             (not value)
```

Comment:
The naming of the keyword is a little strange.

Predicates in Clojure end with `?` and that's enough. In other languages, the var starts with `is`. But in this case we have both.

Is there a way to fix this?

Reply:
@ulisesmac - Yeah, the naming is weird. This is actually from `status-go`
https://github.com/status-im/status-go/blob/4a7d15b36d9c4a298a76db72d23bfd579b3eca67/multiaccounts/settings/columns.go#L395-L399

> Is there a way to fix this?

We can. Changing this would require some effort as it needs to be updated for all clients (in one shot to prevent issues).

Reply:
Hard to do, lets keep the key then